### PR TITLE
Make FormattedAddress component return an EtherscanLink

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

FormattedAddress links throughout the app are currently unintuitive. I expect to be able to click any address and open it in Etherscan. Instead, I have to hover, then click the link in the tooltip.

This PR reworks FormattedAddress to return an EtherscanLink.

Other minor things:
- remove transition from `a` elements.
- remove padding from ClipboardButton
- clean up formatAddress util a bit.

## Screenshots or screen recordings

| before | after |
| --- | --- |
| Screen Shot 2022-08-25 at 2 31 27 PM | Screen Shot 2022-08-25 at 2 30 42 PM |

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -49,14 +50,6 @@ a.quiet {
   }
 }
 
-a.hover-action {
```
Comment (tomquirk) [inline src/styles/index.scss:52]: unused now

File: src/utils/formatAddress.ts [hunk 2]
```diff
@@ -1,21 +1,25 @@
-// Truncates an ETH address to specific number of characters (truncateTo)
-// e.g. truncateEthAddress({
-//   address: '0xf0FE43a75Ff248FD2E75D33fa1ebde71c6d1abAd,
-//   truncateTo: 4
-// }) = '0xf0FE...abAd'
+const DEFAULT_TRUNCATE_TO = 6
```
Comment (tomquirk) [inline src/utils/formatAddress.ts:1]: unrelated cleanup

File: src/styles/index.scss [hunk 3]
```diff
@@ -34,29 +34,13 @@ label {
...
   color: var(--text-action-primary);
   font-weight: 500;
+  transition: none;
 
   &:hover {
     color: var(--text-action-highlight);
   }
 }
 
-a.quiet {
```
Comment (tomquirk) [inline src/styles/index.scss:43]: Don't need it. KILL

**Comment by tomquirk** [comment]
friendly bump @wraeth-eth

**Comment by wraeth-eth** [comment]
minor and out of scope for this one - but we could consider making the tooltip address entirely copyable by click, rather than just the copy button

**Review by wraeth-eth — APPROVED** [review]
LFG
